### DISCUSSION TITLE
Include the datetime that matches cron and triggers a schedule workflow in the context

### DISCUSSION THREAD

**Discussion by dhimmel** [community/community#39078] [Actions]
### Select Topic Area

Product Feedback

### Body

When running a `schedule` workflow on GitHub Actions, the [contexts](https://docs.github.com/en/actions/learn-github-actions/contexts) don't appear to capture the initial datetime that matches a `cron` specification.

Using the following example workflow:

```yaml
on:
  schedule:
    - cron: '51 17 * * *'  # at 17:51 daily
```

Let's assume the first execution of this workflow fails and that the date of the first execution is used as a material parameter for the workflow. When rerunning the failed workflow, the event type will remain as `schedule`, but I can't find any way to determine the initial cron datetime that was matched to trigger the initial build.

In other words, while within a workflow run based on the above `schedule`, is there any way to know which date triggered the cron job?

Noting [`dhimmel/dump-actions-context`](https://github.com/dhimmel/dump-actions-context/actions?query=event:schedule) with the current contexts available to `schedule` workflows.
